noir-projects/aztec-nr/aztec/src/discovery/mod.nr

```diff
@@ -12,9 +11,8 @@ global NOTE_PRIVATE_LOG_RESERVED_FIELDS: u32 = 2;
 
 /// The maximum length of the packed representation of a note's contents. This is limited by private log size and extra
 /// fields in the log (e.g. the combined log and note type ID).
-// TODO (#11634): we're assuming here that the entire log is plaintext, which is not true due to headers, encryption
-// padding, etc. Notes can't actually be this large.
```

Comment:
I think we can close #11634 because PRIVATE_LOG_PLAINTEXT_SIZE_IN_FIELDS is now dynamically derived all the way from `PRIVATE_LOG_SIZE_IN_FIELDS`:
<img width="1326" alt="image" src="https://github.com/user-attachments/assets/277485c8-1094-44f6-982c-c0f7203cac8f" />